scripts/for.sh

```diff
@@ -56,15 +57,48 @@ function shouldRun {
       return 1
     fi
   fi
-  echo $USAGE
+  echo -e "$USAGE"
+  echo -e "$HELP"
   return 1
 }
 
+function getHelp {
+  echo "$USAGE"
+  echo "Available command aliases:\n"
+  IFS="="
+  while read -r name value
+  do
+    echo "${BLUE}${name} : ${NC}${value}\n"
+  done < "node_modules/pathable-scripts/scripts/config/command-aliases.cfg"
+}
+
+function getAliasCommand {
+  command="$1"
+  IFS="="
+  while read -r name value
+  do
+    if [ "$command" == "'$name'" ]; then
+      echo "'${value}'"
+      return 0
+    fi
+  done < "node_modules/pathable-scripts/scripts/config/command-aliases.cfg"
```

Comment:
This looks good, but, shouldn't we encapsulate the commands in separate files? This will be better to manage them

Reply:
Is this in case we want to write complex, multi-line commands? I wonder if thats the right approach, or if we should expose this as a function for other commands to run wathever they choose in the desired repos.
If we do it the alias way, for one, each command will have their entire output repeated in each run, while they may only want to display some of it a single time, and there may be other limitations that im not aware of right now.
Maybe we should consider doing a single CLI for all the scripts, as you suggested for this one, and have a much richer processing there, which we could consider doing in javascript, as i think @faysal-ahmad was porting the deployment scripts to js already.